File: src/features/ai-platform/infra/commands/health.ts
import { invoke } from '@tauri-apps/api/core';

export interface AiPlatformHealth {
  status: string;
  frontendRoot: string;
  backendRoot: string;
  bridgeMode: string;
}

export function getAiPlatformHealth() {
  return invoke<AiPlatformHealth>('ai_platform_health');
}

File: src-tauri/src/ai_platform/interfaces/commands/health.rs
use crate::ai_platform::interfaces::dto::health::AiPlatformHealthDto;

#[tauri::command]
pub fn ai_platform_health() -> AiPlatformHealthDto {
    AiPlatformHealthDto {
        status: "ok".to_string(),
        frontend_root: "src/features/ai-platform".to_string(),
        backend_root: "src-tauri/src/ai_platform".to_string(),
        bridge_mode: "single-cutover-root".to_string(),
    }
}